tests/nested/manual/recovery-system-reboot/task.yaml

```diff
@@ -55,7 +55,7 @@ execute: |
   prev_system=$(remote.exec 'sudo snap recovery' | awk 'NR != 1 { print $1 }')
 
   # create the system
-  change_id=$(post_json_data /v2/systems '{"action": "create", "label": "new-system", "validation-sets": ["test-snapd/test-snapd-pinned-essential-snaps=1"], "mark-default": true, "test-system": true}')
+  change_id=$(post_json_data /v2/systems '{"action": "create", "label": "new-system", "validation-sets": ["test-snapd/test-snapd-pinned-essential-snaps=2"], "mark-default": true, "test-system": true}')
```

Comment:
I think using a sequence number is fine for this case - however I think this change should be explained with a comment of *why* we are chosing sequence 2 here, especially since this is a test.

Reply:
I have added a comment. Tell me if you are OK with it.